fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -0,0 +1,86 @@
+        command << File.expand_path(params[:binary_path])
+        command << "-a #{params[:api_token]}"
+        command << "-p ios"
+        command << File.expand_path(params[:dsym_path])
+
+        return Actions.sh command.join(" ")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Upload dSYM symbolication files to Crashlytics"
+      end
+
+      def self.details
+        [
+          "This action allows you to upload additional symbolication files to Crashlytics.",
+          "It's extra useful if you use it to download the latest dSYM files from Apple when you",
+          "use Bitcode"
+        ].join(" ")
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :dsym_path,
+                                       env_name: "FL_UPLOAD_SYMBOLS_TO_CRASHLYTICS_DSYM_PATH",
+                                       description: "Path to the DSYM file or zip to upload",
+                                       default_value: ENV[SharedValues::DSYM_OUTPUT_PATH.to_s] || (Dir["./**/*.dSYM"] + Dir["./**/*.dSYM.zip"]).first,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Couldn't find file at path '#{File.expand_path(value)}'") unless File.exist?(value)
+                                         UI.user_error!("Symbolication file needs to be dSYM or zip") unless value.end_with?("dSYM.zip") || value.end_with?(".dSYM")
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :api_token,
+                                       env_name: "CRASHLYTICS_API_TOKEN",
+                                       optional: true,
+                                       description: "Crashlytics Beta API Token",
+                                       verify_block: proc do |value|
+                                         UI.user_error!("No API token for Crashlytics given, pass using `api_token: 'token'`") if value.to_s.length == 0
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :binary_path,
+                                       env_name: "FL_UPLOAD_SYMBOLS_TO_CRASHLYTICS_BINARY_PATH",
+                                       description: "The path to the upload-symbols file",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Couldn't find file at path '#{value}'") unless File.exist?(value)
+                                       end)
+        ]
+      end
+
+      def self.output
+        nil
+      end
+
+      def self.return_value
+        nil
+      end
+
+      def self.authors
+        ["KrauseFx"]
+      end
+
+      def self.is_supported?(platform)
+        platform == :ios
```

Comment:
Anything I need to do differently for those platforms? Right now `ios` is hard coded in the generated command. If you can give me example calls for the other platforms I can add them.